stdlib/public/runtime/ProtocolConformance.cpp

```diff
@@ -373,7 +373,7 @@ searchInConformanceCache(const Metadata *type,
     // For generic and resilient types, nondependent conformances
     // are keyed by the nominal type descriptor rather than the
     // metadata, so try that.
-    auto *description = type->getNominalTypeDescriptor().get();
+    auto description = type->getNominalTypeDescriptor().get();
```

Comment:
Does `const auto *` work for these? I think we want to avoid downgrading qualifiers on `auto`.